Cargo.toml

```
@@ -28,5 +28,8 @@ serde_json = "1.0"
 termcolor = "1.0.4"
 toml = "0.5.2"
 
+[dev-dependencies]
+proc-macro-test = { path = "./proc-macro-test" }
```

This needs to be an actual crate with a version number or else `cargo publish` won't know how to treat it when publishing trybuild. Please change to:

```suggestion
trybuild-test-suite = { version = "0", path = "tests/macro" }
```

---

Done